src/PrestaShopBundle/Resources/config/services/adapter/product/_image.yml

```diff
@@ -58,7 +58,10 @@ services:
     arguments:
       - '@doctrine.dbal.default_connection'
       - '%database_prefix%'
+      - "@=service('prestashop.adapter.legacy.context').getLanguage().id"
```

Comment:
You are willing to inject the default language here, so you should get the `PS_LANG_DEFAULT` configuration not the value from context